pyqmix_backend/app.py

```diff
@@ -292,8 +292,8 @@ def standardize_syringe_parameter(pump_id):
         pump.set_volume_unit(prefix='milli', unit='litres')
 
         # Need this as an input instead!
-        pump.set_syringe_params(inner_diameter_mm=23.0329,
-                                max_piston_stroke_mm=60)
+        # How do I get the list of available syringe sizes?
```

Comment:
```python
from pyqmix.pump import syringes
syringe_types = list(syringes.keys())  # Python 3 returns a `dict_keys` object, not a list
```

Reply:
(Oh, and feel free to lecture me on what the "normal procedure" is in GitHub - I am really just asking since I'm unsure what "one would do") :o)

Reply:
Ah! I GET IT! So, what happended was that during my second push I submitted a bit of code that was related to a different issue. Sorry! My head is everywhere this morning. I'll just change the code as per your suggestion, submit it, and then create a separate pull request for the remainder of the issue #6.